MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -461,7 +479,56 @@ internal void FlushIfNeeded()
         /// <param name="color">A color mask.</param>
 		public void Draw (Texture2D texture, Vector2 position, Rectangle? sourceRectangle, Color color)
 		{
-			Draw (texture, position, sourceRectangle, color, 0f, Vector2.Zero, 1f, SpriteEffects.None, 0f);
+			CheckValid(texture);
+            
+			var item = _batcher.CreateBatchItem();
+			item.Texture = texture;
+            
+            // set SortKey based on SpriteSortMode.
+            switch (_sortMode)
+            {
+                // Comparison of Texture objects.
+                case SpriteSortMode.Texture:
+                    item.SortKey = texture.SortingKey;
+                    break;
+                // Comparison of Depth
+                case SpriteSortMode.FrontToBack:
+                    item.SortKey = 0;
+                    break;
+                // Comparison of Depth in reverse
+                case SpriteSortMode.BackToFront:
+                    item.SortKey = 0;
+                    break;
+            }
+
+            Vector2 size;
+
+            if (sourceRectangle.HasValue)
+            {
+                _tempRect = sourceRectangle.Value;                   
```

Comment:
I don't think so. I am going to remove it and use a local temp.
Although  if I do the same for _texCoordTL & _texCoordBR we might get a small hit because C# always initialize variables to their default value.